File: c/c99/1 - Iniciante/1018_cedulas.c
#include <stdio.h>

int main() {
    int valor, nota;
    int notas[] = {100, 50, 20, 10, 5, 2, 1};
    scanf("%d", &valor);
    printf("%d\n", valor);  // valor original
    for (int i = 0; i < 7; i++) {
        nota = valor / notas[i];
        printf("%d nota(s) de R$ %d,00\n", nota, notas[i]);
        valor = valor % notas[i];
    }
    return 0;
}

File: python/1 - Iniciante/1018_cedulas.py
N = int(input())
print(N)
notas = [100, 50, 20, 10, 5, 2, 1]
for nota in notas:
    qntd_notas = N//nota
    print(f'{qntd_notas} nota(s) de R$ {nota},00')
    N %= nota
